cmd/internal/cli/key_import.go

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) 2017-2019, Sylabs Inc. All rights reserved.
```

Comment:
I think the going thing is to put the `[year-when-the-file-was-born]-[current-year]`...
Honestly, I prefer to do it like you did, but for consistency, it should probably be like:
```
// Copyright (c) 2019, Sylabs Inc. All rights reserved.
```